addon/components/one-way-select.js

```diff
@@ -4,10 +4,12 @@ import layout from '../templates/components/one-way-select';
 import { invokeAction } from 'ember-invoke-action';
 
 const {
+  A,
```

Comment:
Small nitpick, but I like renaming this to `emberArray` since `A` is a little too short and unintuitive